internal/app/siftool/modif_test.go

```diff
@@ -21,11 +21,10 @@ func TestApp_New(t *testing.T) {
 		t.Fatalf("failed to create app: %v", err)
 	}
 
-	tf, err := os.CreateTemp("", "sif-test-*")
+	tf, err := os.CreateTemp(t.TempDir(), "sif-test-*")
```

Comment:
isn't the intention of the linting here that we _replace_ os.CreateTemp with t.TempDir?

Given that "Each subsequent call to t.TempDir returns a unique directory", there's no real need for nesting to avoid overlap?

Reply:
Yeah, good point. Have updated the code to just craft a path directly using `t.TempDir()` as the base path.